lib/srv/desktop/rdp/rdpclient/src/rdpdr.rs

```diff
@@ -2129,6 +2139,24 @@ pub struct DeviceIoRequest {
 }
 
 impl DeviceIoRequest {
+    // Used in tests
+    #[allow(dead_code)]
```

Comment:
```suggestion
    #[cfg(test)]
```

Does this have the same effect?

Reply:
2be583cc11860ad93b42d99d5ee0e2a3e35fe563